### PR TITLE
Use local rti in DistributedStopDecentralizedContainerized.lf

### PR DESCRIPTION
**Pull request by lhstrh**

### CODE REVIEW

File: test/C/src/docker/federated/DistributedStopDecentralizedContainerized.lf [hunk 1]
```diff
@@ -6,7 +6,7 @@
 target C {
   coordination: decentralized,
   docker: {
-    rti-image: "rti:latest"
+    rti-image: "rti:local"
```
Comment (edwardalee) [inline test/C/src/docker/federated/DistributedStopDecentralizedContainerized.lf:9]: What does `ti:local` mean?  I couldn't find any documentation for this.  There is no mention here: https://www.lf-lang.org/docs/reference/containerized-execution

Note that searching for "docker" on the lf-lang website give this:

image

Similarly, a search for rti gives no results.

**Review by edwardalee — CHANGES_REQUESTED** [review]
Not clear what this means.

File: test/C/src/docker/federated/DistributedStopDecentralizedContainerized.lf (hunk 1 above)
Comment (lhstrh) [reply test/C/src/docker/federated/DistributedStopDecentralizedContainerized.lf:9]: We index the documentation weekly, so the description isn't indexed yet. The relevant PR is https://github.com/lf-lang/lf-lang.github.io/pull/227.

For Docker users, `rti:local` is very clear. The naming convention for image names is `<user-name>/<image-name>:<tag-name>`. So I simply reserved the `local` tag to refer to a locally built container (i.e., not pulled from Docker Hub).

Comment (lhstrh) [reply test/C/src/docker/federated/DistributedStopDecentralizedContainerized.lf:9]: Actually, I would expect the "Docker" keyword to produce hits in Algolia, but it doesn't. So it seems we have a search bug.

Comment (lhstrh) [reply test/C/src/docker/federated/DistributedStopDecentralizedContainerized.lf:9]: See https://github.com/lf-lang/lf-lang.github.io/issues/229 and https://github.com/lf-lang/lf-lang.github.io/pull/230.

**Pull request merged**